tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -235,7 +271,19 @@ def test_data_duplicate(nhsd_apim_proxy_url, property, pointer, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_data_too_few_items(nhsd_apim_proxy_url, property, pointer, correlation_id):
     """
-    .. py:function:: Test no messages defined
+    .. py:function:: Scenario: A user submitting a request with too few attributes \
+        in the request body receives a 400 'Invalid Value' response
+
```

Comment:
Needs table with props we check for too few items

Reply:
Addressed in request_properties.rst, linked below given/when/then